### PR TITLE
freeswitch 1.4.20

### PR DESCRIPTION
**Pull request by L2G**
* New upstream version
* Add ldns dependency (and re-enable build of mod_enum)
* Add --with-* and --without-* options to customize installed sounds
* Change bootstrap script's "-j<n>" option to a simple "-j" (otherwise it will ignore the option and not put jobs in the backround at all)

### CODE REVIEW

File: Library/Formula/freeswitch.rb [hunk 1]
```diff
@@ -19,20 +19,24 @@ class Freeswitch < Formula
...
+  depends_on "jpeg"
+  depends_on "ldns"
   depends_on "openssl"
   depends_on "pcre"
   depends_on "speex"
   depends_on "sqlite"
 
   def install
-    system "./bootstrap.sh", "-j#{ENV.make_jobs}"
+    system "./bootstrap.sh", "-j"
```
Comment (DomT4) [inline Library/Formula/freeswitch.rb]: Is the `-j` doing anything without `#{ENV.make_jobs}`?

Comment (L2G) [inline Library/Formula/freeswitch.rb]: It does shave about a minute off of the build time on my MacBook Pro.

Comment (DomT4) [inline Library/Formula/freeswitch.rb]: `#{ENV.make_jobs}` should do that as well, I'd presume?

Comment (L2G) [inline Library/Formula/freeswitch.rb]: When I ran `./bootstrap.sh -j8` manually, I got an error message saying that the option wasn't recognized. When I looked at the script, it didn't look like it was expecting a number after `-j` at all. Running `./bootstrap.sh -j` worked and gave a measurable improvement over plain `./bootstrap.sh`.

Comment (dunn) [inline Library/Formula/freeswitch.rb]: Yeah, looks like that's true: https://freeswitch.org/stash/projects/FS/repos/freeswitch/browse/bootstrap.sh#22

**Comment by dunn** [comment]
Were you planning on updating this PR to deal with #41833 at the same time, or should we merge this now?

**Comment by L2G** [comment]
@dunn This can be merged now. If I thought I'd have the other one ready within a day, I'd say wait for it, but I'm not feeling that.

**Comment by dunn** [comment]
Merged in 861f5a4eaae235432c86243b535afc600bcfccce, thanks so much!

**Pull request closed without merging**